src/sst/core/configGraph.cc

```diff
@@ -715,8 +716,7 @@ ConfigGraph::addLink(ComponentId_t comp_id, const std::string& link_name, const
     }
     ConfigLink &link = links[link_names[link_name]];
     if ( link.current_ref >= 2 ) {
-        cout << "ERROR: Parsing SDL file: Link " << link_name << " referenced more than two times" << endl;
-        exit(1);
+        output.fatal(CALL_INFO,1,"ERROR: Parsing SDL file: Link %s referenced more than two times\n",link_name.c_str());
```

Comment:
Nitpicky - but this error should probably say "referenced multiple times" or "referenced more than once".

Reply:
A link needs to be referenced two times, one on each “end”.  This catches the case where it tries to connect to a third (or higher, though it would fail before getting to four) port, thus more than two times,

Reply:
Oops, misread that. Marking this resolved.